packages/liveblocks-react/src/factory.tsx

```diff
@@ -210,23 +242,50 @@ type RoomContextBundle<
   ];
 
   /**
-   * Returns an object that lets you get information about all the the users currently connected in the room.
+   * Returns an object that lets you get information about all the the users
+   * currently connected in the room.
    *
    * @example
    * const others = useOthers();
    *
    * // Example to map all cursors in JSX
-   * {
+   * return (
```

Comment:
I'm really nitpicking but... should we wrap in a fragment to be valid JSX?

Reply:
Returning an array-of-JSX elements is also valid/legal JSX, so I think the example was correct. Nevertheless, I agree wrapping it in a fragment may be nicer to stress the JSX nature of the example 👍 

Done in 9d7fdb25a79357c849daabd30ce2a45d6ae1d48b.